src/scheduleBot_Manager_TGAPI.py

```diff
@@ -80,24 +89,23 @@ def __init__(self, curr_path, config_path, config_private_path):
             message.chat.title
         )
 
-        xls_response = self.manager.xls.find_timetable(response_arr[0], dict())
-        if not response_arr[0] or xls_response == dict():
+        xls_response = self.manager.xls.find_timetable(group, dict())
+        if not group or xls_response == dict():
```

Comment:
To check if something is dict/str/int...etc. it's better to use `isinstance(xls_response, dict) `

Reply:
no, i need to check if this is an empty dict